storage/diemdb/src/transaction_store/mod.rs

```diff
@@ -116,6 +116,13 @@ impl TransactionStore {
         })
     }
 
+    /// Get the first version that txn starts existent.
+    pub fn get_first_txn_version(&self) -> Result<Option<Version>> {
+        let mut iter = self.db.iter::<TransactionSchema>(Default::default())?;
+        iter.seek_to_first();
+        iter.next().map(|res| res.map(|(v, _)| v)).transpose()
```

Comment:
The TxnInfo being smaller, is it fair to say that table can be faster to seek? :P

Reply:
...I don't think there is a big difference since it is binary search + linear at block level.

Reply:
Yeah, I'm interested to know if this is true, too :D

Reply:
Yeah no big deal. Virtually no difference at all. It's just a slightly larger kv being loaded and deserialized.